### PR TITLE
Victor Kalinkin object-manipulation (initial submission)

### PR DESCRIPTION
**Pull request by vkalinkin**
object manipulation 1

### CODE REVIEW

File: object-manipulation-1/src/get-keys.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* exported getKeys */
+function getKeys(object) {
+  var keys = [];
+  var count = 0;
+  for (var key in object) {
+    keys[count] = key;
+    count++;
```
Comment (scbowler) [inline object-manipulation-1/src/get-keys.js:7]: Using `.push` here is a better and safer option

File: object-manipulation-1/src/get-values.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/* exported getValues */
+function getValues(object) {
+  var values = [];
+  var count = 0;
+  for (var value in object) {
+    values[count] = object[value];
+    count++;
```
Comment (scbowler) [inline object-manipulation-1/src/get-values.js:7]: `.push` is a better option here

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking really good! 2 small changes needed. Let me know if you have any questions.

**Comment by vkalinkin** [comment]
object manipulation 1 correction

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**